subsys/bluetooth/controller/hci/hci.c

```diff
@@ -5759,51 +5757,24 @@ int hci_iso_handle(struct net_buf *buf, struct net_buf **evt)
 			return -EINVAL;
 		}
 
-		/* Get free node tx */
-		tx = ll_iso_tx_mem_acquire();
-		if (!tx) {
-			BT_ERR("ISO Tx Buffer Overflow");
-			data_buf_overflow(evt, BT_OVERFLOW_LINK_ISO);
-			return -ENOBUFS;
-		}
-
-		struct pdu_bis *pdu = (void *)tx->pdu;
-
-		/* FIXME: Update to use correct LLID for BIS and CIS */
-		switch (bt_iso_flags_pb(flags)) {
-		case BT_ISO_SINGLE:
-			pdu->ll_id = PDU_BIS_LLID_COMPLETE_END;
-			break;
-		default:
-			ll_iso_tx_mem_release(tx);
-			return -EINVAL;
-		}
-
-		pdu->len = slen;
-		memcpy(pdu->payload, buf->data, slen);
+		/* FIXME: convey group start */
+		sdu_frag_tx.grp_ref_point = 0;
 
+		/* FIXME: temporary interface to enable ISOAL data Tx
+		 * Create provide proper interface between client
+		 * (using ISOAL target_event) and ISOAL, preferably
+		 * without dependence on peeking at LL data.
+		 * Problem is that client must specify a value greater
+		 * than LL bisPayloadCounter or no data is sent.
+		 */
 		struct lll_adv_iso *lll_iso;
 
 		lll_iso = &adv_iso->lll;
+		sdu_frag_tx.target_event = (lll_iso->payload_count / lll_iso->bn);
 
-		uint64_t pkt_seq_num;
-
-		pkt_seq_num = lll_iso->payload_count / lll_iso->bn;
-		if (((pkt_seq_num - stream->pkt_seq_num) & BIT64_MASK(39)) <=
-		BIT64_MASK(38)) {
-			stream->pkt_seq_num = pkt_seq_num;
-		} else {
-			pkt_seq_num = stream->pkt_seq_num;
-		}
-
-		tx->payload_count = pkt_seq_num * lll_iso->bn;
-
-		stream->pkt_seq_num++;
-
-		if (ll_iso_tx_mem_enqueue(handle, tx, NULL)) {
-			BT_ERR("Invalid ISO Tx Enqueue");
-			ll_iso_tx_mem_release(tx);
-			return -EINVAL;
+		/* Start Fragmentation */
+		if (isoal_tx_sdu_fragment(stream->dp->source_hdl, &sdu_frag_tx)) {
+		  return -EINVAL;
```

Comment:
Remove the extra spaces after tabs.

```suggestion
		return -EINVAL;
```